### DISCUSSION TITLE
How can I use bash script in supabase storage?

### DISCUSSION THREAD

**Discussion by tossef** [supabase/supabase#8110] [Questions]
how can I use bash script like "'https://fptgcgirjfstyhzagrob.supabase.co/rest/v1/candidates_profiles?select=serial_no'" for my storage.
For example my storage has file/documents/file.pdf.
I specifically want to download these documents how can I do that with above mentioned script?
Thank you in advance.

**Comment by GaryAustin1** [comment]
I assume you mean access the storage API directly.
https://supabase.github.io/storage-api/
From a link in the storage guide in docs https://supabase.com/docs/guides/storage.

supabaseurl/storage/v1/   goes in front of those.
